support/k8s/k8s-workload-registrar/controllers/utils.go

```diff
@@ -0,0 +1,138 @@
+/*
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package controllers
+
+import (
+	"context"
+	"encoding/binary"
+	"fmt"
+	"hash"
+	"hash/fnv"
+
+	"github.com/davecgh/go-spew/spew"
+	spiffeidv1beta1 "github.com/spiffe/spire/api/spiffecrd/v1beta1"
+	"github.com/spiffe/spire/proto/spire/api/registration"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/util/rand"
+	clientgoscheme "k8s.io/client-go/kubernetes/scheme"
+	ctrl "sigs.k8s.io/controller-runtime"
+)
+
+func NewManager() (ctrl.Manager, error) {
+	scheme := runtime.NewScheme()
+	_ = clientgoscheme.AddToScheme(scheme)
+	_ = spiffeidv1beta1.AddToScheme(scheme)
+	_ = corev1.AddToScheme(scheme)
+
+	mgr, err := ctrl.NewManager(ctrl.GetConfigOrDie(), ctrl.Options{
+		MetricsBindAddress: "0",
+		Scheme:             scheme,
+		Port:               9443,
```

Comment:
Is this likely to change? does it need to be configurable?

Reply:
It might, kubebuilder has it configurable. I made the change here, remove the Port because we don't have a webhook in here yet: https://github.com/spiffe/spire/pull/1581/commits/137f7dd0c496939fa904911c66b63961cf567ce2